shared/whatYouNeed/kitra520.md

```diff
@@ -0,0 +1,15 @@
+<img style="float: right;padding-left: 10px;" src="/img/kitra/710.png" width="35%">
+
+
+* [A Kitra520][kitra520-link] product accelerator development board from [RushUp](http://www.rushup.tech/).
+* A micro USB cable.
+* **[Optional]** A [2A USB micro power supply][psu]
+* At least one of Wireless communication antennas, the kit should contain 1 antenna.
+* [Samsung ARTIK 5][artik5-spec-link] development board. We recommended either version 3.0-3.3 or version 0.5.
```

Comment:
It might be worth explaining why we need this board as well, and that its only for the provisioning phase